roachtest: import/tpcc/warehouses=1000/nodes=4 failed

SHA: https://github.com/cockroachdb/cockroach/commits/031748232a601aab191fd3fb00dc298ca20e4506

Parameters:

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stressrace instead of stress and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
make stress TESTS=import/tpcc/warehouses=1000/nodes=4 PKG=roachtest TESTTIMEOUT=5m STRESSFLAGS='-stderr=false -maxtime 20m -timeout 10m'
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=932135&tab=buildLog

```
The test failed on master:
	test.go:779: test timed out (5h0m0s)
	test.go:498,cluster.go:890,import.go:43,cluster.go:1199,errgroup.go:58: /home/agent/work/.go/bin/roachprod run teamcity-932135-import-tpcc-warehouses-1000-nodes-4:1 -- ./workload fixtures make tpcc --warehouses=1000 --csv-server='http://localhost:8081' --gcs-bucket-override=cockroach-tmp --gcs-prefix-override=teamcity-932135-import-tpcc-warehouses-1000-nodes-4 returned:
		stderr:
		
		stdout:
		Connection to 35.232.31.125 closed by remote host.
		Error:  exit status 255
		: exit status 1
	test.go:498,cluster.go:1220,import.go:46,import.go:56: Goexit() was called

```

Also failed on [32 nodes](https://github.com/cockroachdb/cockroach/pull/27104), FWIW.


```
health: health check terminated with dial tcp 35.185.43.135:26257: connect: connection refused
> /home/tschottdorf/go/bin/roachprod get tobias-1538822415-import-tpcc-warehouses-1000-nodes-32 logs artifacts/import/tpcc/warehouses=1000/nodes=32/logs
tobias-1538822415-import-tpcc-warehouses-1000-nodes-32: getting (scp) logs artifacts/import/tpcc/warehouses=1000/nodes=32/logs
......................................[ 302] import/tpcc/warehouses=1000/nodes=32: retrieving logs (38s)
............................................................[ 303] import/tpcc/warehouses=1000/nodes=32: retrieving logs (1m38s)
.....................OK
--- FAIL: import/tpcc/warehouses=1000/nodes=32 [unstable] (18211.61s)
        test.go:819: test timed out (5h0m0s)
        test.go:500,cluster.go:897,import.go:50,cluster.go:1219,errgroup.go:58: /home/tschottdorf/go/bin/roachprod run tobias-1538822415-import-tpcc-warehouses-1000-nodes-32:1 -- ./workload fixtures make tpcc --warehouses=1000 --csv-server='http://localhost:8081' --gcs-bucket-override=cockroach-tmp --gcs-prefix-override=tobias-1538822415-import-tpcc-warehouses-1000-nodes-32 returned:
                stderr:
                Connection to 35.196.227.113 closed by remote host.
                Error:  exit status 255

                stdout:
                : exit status 1
        test.go:500,cluster.go:1240,import.go:53,import.go:63: dial tcp 35.185.43.135:26257: connect: connection refused
```

@danhhz can you prioritize looking into this? Makes me nervous to have this fail for over a week.

I'm also going to kick of another one and keep the cluster around in case it's something obvious I can spot.

--- a/pkg/ccl/workloadccl/fixture.go
+++ b/pkg/ccl/workloadccl/fixture.go
@@ -128,7 +128,8 @@ func GetFixture(
 ) (Fixture, error) {
 	var fixture Fixture
 	var err error
-	for r := retry.StartWithCtx(ctx, retry.Options{MaxRetries: 10}); ; r.Next() {
+	var notFound bool
+	for r := retry.StartWithCtx(ctx, retry.Options{MaxRetries: 10}); r.Next(); {
 		err = func() error {
 			b := gcs.Bucket(config.GCSBucket)
 			if config.BillingProject != `` {
@@ -138,6 +139,7 @@ func GetFixture(
 			fixtureFolder := generatorToGCSFolder(config, gen)
 			_, err := b.Objects(ctx, &storage.Query{Prefix: fixtureFolder, Delimiter: `/`}).Next()
 			if err == iterator.Done {
+				notFound = true
 				return errors.Errorf(`fixture not found: %s`, fixtureFolder)
 			} else if err != nil {
 				return err
@@ -159,7 +161,7 @@ func GetFixture(
 			}
 			return nil
 		}()
-		if err == nil {
+		if err == nil || notFound {
 			break
 		}
 	}

